hs/CssGen.hs

```diff
@@ -591,9 +591,9 @@ tdColours = ".timetable " ?  do
     td # ("in-conflict" *= "false") # ("satisfied" *= "true") # ("type" *= "L") ? do
             backgroundColor blue3
     td # ("in-conflict" *= "false") # ("satisfied" *= "true") # ("type" *= "T") ? do
-            backgroundColor teal1
```

Comment:
Rather than repeat the selectors, why don't we just remove the `# ("type *= "L")`.